includes/services/extended/0-google-base.php

```diff
@@ -32,6 +32,7 @@ function __construct() {
 		$this->redirect_uri = $creds['redirect_uri'];
 		$this->key          = $creds['key'];
 		$this->secret       = $creds['secret'];
+		$this->token        = $creds['scope'];
```

Comment:
Without actually testing this, it seems like this should probably be `$this->scope`, no?

Reply:
Oops! Sorry, had two copies of the repository, one of my fork and one of the actual repository. I had fixed this on the normal one after catching during testing but forgot to copy the fix over to my fork. Apologies! It's fixed now.